service/resource/cloudformation/delete.go

```diff
@@ -3,14 +3,76 @@ package cloudformation
 import (
 	"context"
 
+	"github.com/aws/aws-sdk-go/aws"
+	awscloudformation "github.com/aws/aws-sdk-go/service/cloudformation"
+	"github.com/giantswarm/aws-operator/service/key"
+	"github.com/giantswarm/microerror"
 	"github.com/giantswarm/operatorkit/framework"
 )
 
 func (r *Resource) ApplyDeleteChange(ctx context.Context, obj, deleteChange interface{}) error {
+	customObject, err := key.ToCustomObject(obj)
+	if err != nil {
+		return microerror.Mask(err)
+	}
+	deleteStackInput, err := toDeleteStackInput(deleteChange)
+	if err != nil {
+		return microerror.Mask(err)
+	}
+
+	stackName := deleteStackInput.StackName
+	if *stackName != "" {
+		_, err := r.awsClient.DeleteStack(&deleteStackInput)
+
+		if err != nil {
+			return microerror.Mask(err)
+		}
+
+		r.logger.Log("cluster", key.ClusterID(customObject), "debug", "deleted stacks from the AWS API")
+	} else {
+		r.logger.Log("cluster", key.ClusterID(customObject), "debug", "the stacks do not need to be deleted from the AWS API")
```

Comment:
Again in azure-operator they are using these log messages. I think they are more helpful.

- `deleting Azure resource group: deleted`
- `deleting Azure resource group: already deleted`

Reply:
Ok done